zerver/models.py

```diff
@@ -751,6 +759,17 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
         wildcard_mention_policy=int,
     )
 
+    REALM_PERMISSION_GROUP_SETTINGS: Dict[str, GroupPermissionSetting] = dict(
+        create_multiuse_invite_group=GroupPermissionSetting(
+            require_system_group=True,
+            allow_internet_group=False,
+            allow_owners_group=False,
+            allow_nobody_group=True,
+            default_group_name=MODERATORS_GROUP_NAME,
```

Comment:
After discussion with @alya, I think we should change this to `ADMINISTRATORS_GROUP_NAME`; we couldn't come up with a ton of use cases for moderator use of this, and I feel like that might better match user expectations in new organizations.